meson.build

```diff
@@ -357,6 +357,13 @@ if features['libdl']
     dependencies += libdl
 endif
 
+subrandr = dependency('subrandr', version: ['>= 0.1.0', '<= 0.2.0'], required: get_option('subrandr'))
```

Comment:
`<= 0.2.0`? If you don't want for it to work on 0.2, probably should be `< 0.2.0`. Also do you plan to break everything on next version? :)

Reply:
Ah thanks, yeah it probably just shouldn't have any version restriction whatsoever since I probably won't break ABI any time soon and this can't be keyed on ABI version.